src/graphics.rs

```diff
@@ -67,10 +67,10 @@ where
     }
 
     fn fill_solid(&mut self, area: &Rectangle, color: Self::Color) -> Result<(), Self::Error> {
-        let fb_size = self.model.framebuffer_size();
```

Comment:
I'm still conflicted about this. Framebuffer_size is supposed to be "the whole pixel memory" which in case of clear is what we want AFAICS.

Reply:
Clear doesn't use `fill_solid`. At least not with this version. So nothing here is going to affect the clear function.

However I do have an alternative approach. Will add another commit, see if you like that any better.

Reply:
I added a `framebuffer_size` function and use that now, hopefully this will address your concern here.

Reply:
Having said that, there is still the problem that both "whole pixel memory" and "displayable area" both need to start at 0,0. I don't think the concept is very usable as is.